docs/workforce-management-framework/topics/con-wfm-extending-user-interface.adoc

```diff
@@ -0,0 +1,57 @@
+[id='{context}-con-extending-the-user-interface']
+= Extending the User Interface
+
+{Feature} framework provides a set of extension points which allows a developer to modify an Angular User Interface (UI).
+All modifications are done by supplying additional modules which provide a set of the Angular.js directives and are used to extend the Mobile and Portal UI.
+The {Feature} Application is shipped with the example `angularjs-extensions` example that extends the Demo UI:
+
+ * https://github.com/feedhenry-raincatcher/raincatcher-angularjs/tree/master/packages/angularjs-extensions
+
+This Module contains the example directives that are used for demo purposes and it can be used as a template to extend the existing {Feature} UI.
+
+== Extending Workorders
+
+By default, {Feature} provides a minimal set of defined fields for a _workorder_.
+Depending on the needs of the business, a developer can add additional data and UI elements, for example, gps location, an address or a description UI element.
+
+=== Building Workorders
```

Comment:
This is very good example when meaning is changed and no longer makes sense.
Extending Workorders != Budiling Workorders. Can we revert?

Reply:
I will revert this back to origin "=== Building"